xmtp_mls/src/storage/encrypted_store/key_store.rs

```diff
@@ -0,0 +1,38 @@
+use super::DbConnection;
```

Comment:
For consistency I think this file should be called `key_store_entry.rs`

Reply:
Good call